src/Nethermind/Nethermind.Evm/BlockExecutionContext.cs

```diff
@@ -15,14 +15,11 @@ namespace Nethermind.Evm;
     public BlockExecutionContext(BlockHeader blockHeader)
     {
         Header = blockHeader;
-        if (blockHeader?.ExcessBlobGas is not null)
+        if (!BlobGasCalculator.TryCalculateBlobGasPricePerUnit(blockHeader?.ExcessBlobGas ?? 0, out UInt256 blobBaseFeeResult))
```

Comment:
It's seemed better when the code did not do anything if blob fields were not set. I guess it can be archieved by better test initialization.

Reply:
I agree. fixed